Babylonian_Method/Babylonian_Method.go

```diff
@@ -0,0 +1,34 @@
+/* Babylonian method for square root
+  1. Start with an arbitrary positive start value x (the closer to the 
+     root, the better).
+  2. Initialize y = 1.
+  3. Do following until desired approximation is achieved.
+      a) Get the next approximation for root using average of x and y
+      b) Set y = n/x */
+
+package main
```

Comment:
Please change the package name to something meaningful

Reply:
In Go, we have package main as the root package, like it is int main() in C++.,
So I guess it makes sense.